Makefile

```diff
@@ -6,3 +7,19 @@ fmt:
 
 deploy: config
 	firebase deploy
+
+# --- Install software ----
+
+software-update: firebase-update jsonnet-update  ## Install/update tools needed by this Makefile
```

Comment:
not  sure if `software-update` is the best name - this would also work if there wasn't any software there in the first place right? how about `make tools` or `make install-tools` or so?

Reply:
the reason I went for "update" is that is what you'll do more often than install. the firebase cli even changed the other day telling me to "update". I have no strong preference though. But any change I make needs to be do-able individually as `verb-firebase` and `verb-jsonnet` as you may need to do one as root and one not (and firebase is 120MB so you may not want to do that - I haven't yet figured out how to check if up-to-date yet so as to not download something you have).

Reply:
ok, understood... I still think (and maybe that's misguided) it should be optimised for first time users and updating something that's not there is confusing imo.

I'd go with `get-tools` or `install-tools`, but I see your point, so no strong feelings.

Reply:
i've gone with `get-*` - nice and short and does not imply initial install or update. can mean either.

Reply:
done.